src/MailHide.php

```
@@ -32,6 +31,8 @@
  * Render and validate MailHide reCaptchas.
  *
  * @deprecated This class is deprecated and will be removed in version 4.0
+ *
+ * @final This class should not be extended and will be marked final in version 4.0
```

Is this part really necessary, given that the class is already marked as deprecated?

---

For these classes it was giving an error on continuous integration checks, we added that block to suppress the errors and it was suggested here: https://github.com/laminas/laminas-recaptcha/pull/44#discussion_r2466264497

---

The suggestion by @samsonasik is correct, but the description is confusing because the class is already marked as deprecated. Remove the description and use only `@final`.

---

I have removed the description and keep only @final